RELEASE.md

```diff
@@ -1,6 +1,44 @@
-## Release Process
+# OpenZipkin Release Process
 
-* tag the commit `git tag -v0.0.0`
-* push tags to origin `git push origin --tags`
-* Travis will automatically build and push to rubygems
-* TODO: have travis push the tag using info from version.rb
+This repo uses semantic versions. Please keep this in mind when choosing version numbers.
+
+1. **Alert others you are releasing**
+
+   There should be no commits made to master while the release is in progress (about 10 minutes). Before you start
+   a release, alert others on [gitter](https://gitter.im/openzipkin/zipkin) so that they don't accidentally merge
+   anything. If they do, and the build fails because of that, you'll have to recreate the release tag described below.
+
+1. **Push a git tag**
+
+   The tag should be of the format `N.M.L`, ex `git tag 1.18.1; git push origin 1.18.1`.
```

Comment:
1) I don't know what N.M.L stands for but I think it'd be clearer to use semver terminology, e.g. `MAJOR.MINOR.PATCH`

2) The tag commands should be separated by `&&` instead of `;` so it doesn't try to push if the local tag command failed.